doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -75,17 +76,325 @@ This is useful when making a change on the PCB (for example, when changing the c
+
+Datasheet register field
+^^^^^^^^^^^^^^^^^^^^^^^^
+
++---------------------+------------------------------------+-------------------------------+
+| Datasheet field     | JSON variable                      | JSON value                    |
++=====================+====================================+===============================+
+| TEMPINTERVAL (A)    | tempMeasIntervalSeconds            | 5                             |
++---------------------+------------------------------------+-------------------------------+
+| TEMPDELTA (B)       | tempDeltaCalibrationTriggerCelsius | 10                            |
++---------------------+------------------------------------+-------------------------------+
+| INTERVALMAXNO (C)   | maxMeasIntervalBetweenCalibrations | 3                             |
++---------------------+------------------------------------+-------------------------------+
+| ENABLE              | calibrationEnabled                 | -                             |
++---------------------+------------------------------------+-------------------------------+
+
+LFRC Autocalibration is not configured in the JSON configuration files for the DK, as the default values (``4``, ``0.5`` and ``2``) will be good enough for most use-cases.
+However, the example would be as follows:
+
+.. code-block:: json
+
+   {
+     "lfrccal": {
+       "calibrationEnabled": true,
+       "tempMeasIntervalSeconds": 5,
+       "tempDeltaCalibrationTriggerCelsius": 10,
+       "maxMeasIntervalBetweenCalibrations": 3
+     }
+   }
+
+.. note::
+
+   * Use the default values unless a custom calibration profile is needed.
+   * ``tempDeltaCalibrationTriggerCelsius``: In the JSON BICR format, the maximum allowable value for this field is 31.75 °C.
+     Therefore, 31.75 is used in place of 40 to ensure compatibility.
+
+HFXO configuration and startup
+------------------------------
+
+The following table maps configuration options for HFXO modes:
+
++------------------------+-------------------------------+
+| Datasheet register     | JSON variable                 |
+| field                  | (within ``hfxo``)             |
++========================+===============================+
+| HFXO.CONFIG: MODE      | mode                          |
++------------------------+-------------------------------+
+| HFXO.CONFIG: LOADCAP   | builtInLoadCapacitancePf      |
++------------------------+-------------------------------+
+| HFXO.STARTUPTIME: TIME | startupTimeUs*                |
++------------------------+-------------------------------+
+
+(*) Depends on ``"builtInLoadCapacitors": true``
+
+hfxo-mode
+^^^^^^^^^
+
++-----------------+-------------------+
+| Datasheet (MODE)| JSON (mode)       |
++=================+===================+
+| Crystal         | CRYSTAL           |
++-----------------+-------------------+
+
+The corresponding JSON configuration, based on the previous table, is as follows:
```

Comment:
This is not obvious that it is referring back to the Datasheet register field table. Not sure how to make it more obvious, but I think this could use some more clarity.